plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/DeployEnvironmentDelegate.java

```diff
@@ -31,11 +31,11 @@
 
   /**
    * @param project Eclipse project to be deployed
-   * @param stagingDirectory directory where implementors methods should place necessary files for
-   *     deployment, where this job will execute {@code gcloud app deploy}
-   * @param safeWorkDirectory directory path that implementors methods may create safely to use as
+   * @param stagingDirectory directory where implementing methods should place necessary files for
```

Comment:
That's still not normal usage. "methods" is implied here. I think something like "directory where files to be deployed are placed" is enough.